BREAKING.md

```diff
@@ -13,13 +13,21 @@ There are a few steps you can take to write a good change note and avoid needing
 ## 0.56 Breaking changes
 - [`MessageType.Save` and code that handled it was removed](#messageType-save-and-code-that-handled-it-was-removed)
 - [Removed `IOdspResolvedUrl.sharingLinkToRedeem`](#Removed-IOdspResolvedUrl.sharingLinkToRedeem)
+- [Removed `PureDataObject.requestFluidObject_UNSAFE`](#Removed-PureDataObject.requestFluidObject_UNSAFE)
+- [Modified `PureDataObject.getFluidObjectFromDirectory`](#Modified-PureDataObject.getFluidObjectFromDirectory)
 
 ### `MessageType.Save` and code that handled it was removed
 The `Save` operation type was deprecated and has now been removed. This removes `MessageType.Save` from `protocol-definitions`, `save;${string}: ${string}` from `SummarizeReason` in the `container-runtime` package, and `MessageFactory.createSave()` from and `server-test-utils`.
 
 ### Removed `IOdspResolvedUrl.sharingLinkToRedeem`
 The `sharingLinkToRedeem` property is removed from the `IOdspResolvedUrl` interface. The property can be accesed from `IOdspResolvedUrl.shareLinkInfo` instead.
 
+### Removed `PureDataObject.requestFluidObject_UNSAFE`
+The `requestFluidObject_UNSAFE` property is removed from the `PureDataObject`.
```

Comment:
We should add a note here that if they want to continue to use the URI,  they need to call `handleFromLegacyUri`

Reply:
We don't want to encourage the usage here, but yes, let's give it an option. Good callout.